doc/source/parameters/dem/boundary_conditions.rst

```diff
@@ -28,6 +28,9 @@ In this subsection, the boundary conditions of the DEM simulation are defined. F
       set rotational vector x = 1
       set rotational vector y = 0
       set rotational vector z = 0
```

Comment:
Shouldn't we also just change the way we define the rotation the `rotation vector` in the to be the same as the `point on rotation vector`. I find the new way more elegant.

Reply:
The new way is more elegant, but i want to introduce the change progressively. First fix the bug, then refactor the parameters in another PR. I would like to remove all of the x, y , z parameters as individual and always use lists, but I would rather do this in another PR